### DISCUSSION TITLE
I can't edit my profile

### DISCUSSION THREAD

**Discussion by omini-miguel** [community/community#49854] [Other Feature Feedback, Questions, & Ideas] [answered]
a

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @NGMinguall ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
